recipes/scikit-datasets/scikit-datasets/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "scikit-datasets" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-datasets-{{ version }}.tar.gz
+  sha256: c6361bd25d143e3eb652575a2b1e24d17c2c5adf8b5707bd489599df6df54dc6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - numpy
+    - python
```

Need to specify a version for Python:
```suggestion
    - python >=3.8
  run:
    - numpy
    - python >=3.8
```